internal/tuf/tuf.go

```diff
@@ -29,6 +30,9 @@ const (
 	GlobalRuleThresholdType        = "threshold"
 	GlobalRuleBlockForcePushesType = "block-force-pushes"
 	RemoveGlobalRuleType           = "remove"
+
+	PreCommitStageName = "pre-commit"
```

Comment:
Thinking about it more, I'd setup a stage type so we can better validate what we see.

something like

```
type HookStage uint

const (
    PreCommit HookStage = iota
    ...
)

func (h HookStage) String() string {...}
```